tests/pyincore/analyses/mlenabledcgeslc/test_mlcgeslc.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+from pyincore import IncoreClient, globals as pyglobals
+from pyincore.analyses.mlenabledcgeslc import MlEnabledCgeSlc
+
+client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
+
+mlcgeslc = MlEnabledCgeSlc(client)
+
+sector_shocks = "65fdeed7e42f3b0da56c4eef"
```

Comment:
Is it possible to chain it with sector shock? Or you are missing the sector_to_building file?

Reply:
I don't understand the question. We only need sector shocks for this. Capital Shocks analysis needs sector to building file.

Reply:
Ohh, yes it can be chained. I have done that in the Docs PR.